features/support/download_helpers.rb

```diff
@@ -0,0 +1,41 @@
+module DownloadHelpers
```

Comment:
I had not thought about downloading files, so this looks interesting. It would be good if you could go through this with me as I'd like to know how you thought of this solution 👍

Reply:
I just implemented it Tim, all credit belongs to @Toyinccs.